src/SuperGoodRobot/COLLECT_ITEM_1.ino

```diff
@@ -0,0 +1,73 @@
+//starting position
+#define FOREARMSERVO_POSITION_1_A 90//some number
+#define BASESERVO_POSITION_1_A 90//some number
+#define BASE_POSITION_1_A 90//some number
+
+//getting item position
+#define FOREARMSERVO_POSITION_1_B 30//some number
+#define BASESERVO_POSITION_1_B 30//some number
+#define BASE_POSITION_1_B 90//some number
+
+//dropping item position
+#define FOREARMSERVO_POSITION_1_C 90//some number
+#define BASESERVO_POSITION_1_C 90//some number
+#define BASE_POSITION_1_C 90//some number
```

Comment:
Instead of arbitrary "A, B, C" and seemingly useless _1_, why not just call "BASE_POSITION_DROP" or something more intuitive?